### DISCUSSION TITLE
Adicionar WinDump ao repositório

### DISCUSSION THREAD

**Discussion by grillado** [microsoft/winget-pkgs#106537] [General]
Saudações,
É possível adicionar o windump no repositório do winget?
https://www.winpcap.org/windump/install/bin/windump_3_9_5/WinDump.exe
Desculpe se a pergunta parecer idiota, sou novo por aqui..
Obrigado..

**Comment by mdanish-kh** [comment]
I believe WinDump requires WinPcap to be installed first for correct functioning of the package.

> To run WinDump:
>
>   1. Install WinPcap
>   2. Download WinDump.exe
>   3. execute the program from the command line

From https://www.winpcap.org/windump/install/default.htm

The package is blocked by:
- https://github.com/microsoft/winget-cli/issues/163

Note: For future package requests, I would suggest opening a [package request issue](https://github.com/microsoft/winget-pkgs/issues/new/choose) instead of a discussion to better track the request

- Edit: Created an issue for better tracking #107781
